sql/alps_submitter.go

```diff
@@ -46,6 +51,32 @@ type CrossColumn struct {
 	HashBucketSize int
 }
 
+func (nc *NumericColumn) GenerateCode() string {
+	return fmt.Sprintf("tf.feature_column.numeric_column(\"%s\", shape=(%d,))", nc.Key, nc.Shape)
+}
+
+func (bc *BucketColumn) GenerateCode() string {
+	return fmt.Sprintf(
+		"tf.feature_column.bucketized_column(%s, boundaries=%s)",
+		bc.SourceColumn.GenerateCode(),
+		strings.Join(strings.Split(fmt.Sprint(bc.Boundaries), " "), ","))
```

Comment:
Why not just `fmt.Sprint(bc.Boundaries)`?

Reply:
The `fmt.Sprint` results a string with space separator which I want it to be a comma .